libmscore/property.cpp

```diff
@@ -169,6 +169,10 @@ static const PropertyData propertyList[] = {
       { P_ID::LASSO_SIZE,          false, 0,               P_TYPE::SIZE_MM  },
 
       { P_ID::TIME_STRETCH,        false, 0,               P_TYPE::REAL     },
+      { P_ID::ORNAMENT_STYLE,      false, "ornamentStyle", P_TYPE::ORNAMENT_STYLE},
```

Comment:
As stated in the comment at the start of the array, the first element here needs to be in the exact same order than the enum class P_ID in property.h.